stock_inventory_preparation_filter/models/stock_inventory.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2015 AvanzOSC - Oihane Crucelaegi
+# Copyright 2015 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class StockInventoryEmptyLines(models.Model):
+    _name = 'stock.inventory.line.empty'
+
+    product_code = fields.Char(
+        string='Product Code',
+        required=True,
+    )
+    product_qty = fields.Float(
```

Comment:
The idea behind this when we did it is to not put any kind of precision as this a "free table" to input data. Then when this is moved to the inventory lines, precision is applied.

Reply:
IMHO, that is against the principle of decimal precision and can be weird for user... Do you want to say that if a user enter 1.333333333 in field, precision is 2 decimals, so inventory get 1.33 but in empty lines (in notebook page 'Capture Lines') he will see 1.333333333?

For me, that's inconsistent

Reply:
Well, in fact the user will see 1.33 as well because by default in screen they are presented with 2 decimals, but the rest will be stored with full precision. The idea behind this is to save as is the numbers introduced, but rip them when validating. Why? Because you can follow the trace in the DB of original values vs used values.

Reply:
That's inconsistent for me. I know about what is stored in DB is different from what is displayed.

But if 'Product Unit of Measure' precision is 3, you have to display 1.333 !